### ISSUE TITLE
Bot asks for location information

### ISSUE THREAD

**Issue by purrloftruth** [erpheus/hideit-bot#7]
After a recent telegram update, I’m receiving this pop up when trying to use hideitbot.

![994D9293-BD32-4E26-9C90-57A2C6FDB138](https://user-images.githubusercontent.com/280610/84814977-4eee5a00-afe0-11ea-8ee1-218f2160f043.jpeg)

Is there an explanation for why sending my location to a third party is necessary to hide my messages?

**Comment by erpheus** [comment]
I haven’t actually updated anything in the bot in years. The code that is running is the same as the one that appears in this repo, and there is nothing about location.

It must have been some telegram update bringing some sort of default configuration, but it is quite weird. I’ll look into it ASAP but currently I’m very busy, sorry.

**Comment by erpheus** [comment]
I cleaned up a bit in the server and restarted the bot and it seems to be working again. I think this message was caused by telegram assuming wrongly that the bot was asking for those permissions instead of simply not responding?

I'm not entirely sure, but after restarting it I'm not getting this prompt either. I'll leave the issue open as I'll investigate into this in the future to make sure that, even if it fails, no such message is displayed.

In any case, thanks a lot for the report!
